recipes/maddy/all/test_package/example.cpp

```diff
@@ -0,0 +1,24 @@
+/*
+ *  This example was copied from https://github.com/progsource/maddy/blob/master/README.md
+ */
+
+#include <memory>
```

Comment:
My suggestions to the test package are completely superfluous, but it's nice to see some output imho.
My changes are small.
```suggestion
#include <iostream>
#include <memory>
```